httptransport/client/differ.go

```diff
@@ -0,0 +1,162 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"path"
+	"strings"
+	"sync"
+
+	"github.com/google/uuid"
+	"github.com/quay/claircore/libvuln/driver"
+
+	"github.com/quay/clair/v4/httptransport"
+	"github.com/quay/clair/v4/matcher"
+)
+
+var _ matcher.Differ = (HTTP)(nil)
+
+// DeleteUpdateOperations attempts to delete the referenced update operations.
+func (c HTTP) DeleteUpdateOperations(ctx context.Context, ref ...uuid.UUID) error {
+	var wg sync.WaitGroup
+	errs := make([]error, len(ref))
+	wg.Add(len(ref))
+	u, err := c.addr.Parse(httptransport.UpdatesAPIPath)
+	if err != nil {
+		return err
+	}
+
+	for i, lim := 0, len(ref); i < lim; i++ {
```

Comment:
Since this is HTTP, and it'll directly use up conns, we probably want either a MAX_REFs which returns an error, or a way to bound that concurrency.